### PR TITLE
Basic Physics Calculator

### PR DESCRIPTION
**Pull request by RiyaJain12**
A calculator that calculates the basics physics quantities like speed,acceleration,velocity ,force etc. It also has the functionality that it can give the answer in  any unit we wanted to get.It also has the side navbar for the direct move to the page that has the parameter we wants to find.

## Description 📜

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.
Fixes # 236

<hr>
 
## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/vasu-1/CalcHub/blob/main/.github/ContributingGuidelines.md) & [Code of conduct](https://github.com/vasu-1/CalcHub/blob/main/.github/CODE_OF_CONDUCT.MD) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GSSOC'22 contributor
- [x] Have you added a working video of your calculator ?

<hr>

## Screenshots / Gif (Optional) 📸
Screenshot (180)

<hr>

### CODE REVIEW

**Review by mansi-garg-12 — CHANGES_REQUESTED** [review]
Code is good to go...
But try to make it more responsive for phone, tablet 
Example size of input box is too small in case of phone and tablet.

File: Calculators/Basic_physics calculator/styles.css [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (vasu-1) [inline Calculators/Basic_physics calculator/styles.css:193]: remove unnecessary lines !!

**Review by vasu-1 — CHANGES_REQUESTED** [review]
Make it responsive !!

add this following line in the head tag
`<meta name="viewport" content="width=device-width, initial-scale=1.0" />`

and try to make is responsive !

suggestion :  use bootstrap

**Comment by RiyaJain12** [comment]
@vasu-1 
done!

File: Calculators/Basic_physics calculator/styles.css [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
```
Comment (vasu-1) [inline Calculators/Basic_physics calculator/styles.css:2]: remove these lines too

**Review by vasu-1 — CHANGES_REQUESTED** [review]
Remove `.vscode/settings.json` from pr!

And yeah, as shown in below image, that drawer is not visible in mobile screen mode!
can you just fix it ?
image

**Comment by RiyaJain12** [comment]
@vasu-1 
done all the changes!

**Review by vasu-1 — APPROVED** [review]
LGTM @RiyaJain12

**Pull request merged**